esbuild.config.js

```diff
@@ -0,0 +1,16 @@
+import esbuildServe from "esbuild-serve";
+
+esbuildServe(
+  {
+    entryPoints: ["./base/GoldenSun.ts"],
+    bundle: true,
+    minify: true,
+    sourcemap: "inline",
+    target: ["es2019", "chrome58", "firefox57"],
```

Comment:
I don't think we need `chrome58` and `firefox57` targets. Is there a reason for them?